presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestHistogram.java

```diff
@@ -471,16 +423,20 @@ private void testSharedGroupByWithOverlappingValuesRunner(InternalAggregationFun
         test1.runPagesOnAccumulatorWithAssertion(0L, test1.createGroupedAccumulator(), aggregationTestOutput1);
     }
 
-    private InternalAggregationFunction getInternalAggregationFunctionVarChar(HistogramGroupImplementation groupMode)
+    private InternalAggregationFunction getInternalDefaultAggregationn()
```

Comment:
good question. because I wanted to be able to change it and bind the method less to what it creates. OTOH, knowing that might help.  it is a unit test, so i try not to overthink it